Sources/Apodini/Request/DynamicProperty.swift

```diff
@@ -0,0 +1,208 @@
+                    assert(((try? typeInfo(of: property.type).kind) ?? .none) == .struct, "\(Target.self) \(property.name) on element \(info.name) must be a struct")
+                    
+                    mutation(&t, property.name)
+                    try property.set(value: t, on: &self)
+                case var dp as DynamicProperty:
+                    assert(((try? typeInfo(of: property.type).kind) ?? .none) == .struct, "DynamicProperty \(property.name) on element \(info.name) must be a struct")
+                    
+                    dp.apply(mutation)
+                    try property.set(value: dp, on: &self)
+                case var dyn as Dynamics:
+                    // no assertion needed because Dynamics is defined by Apodini
+                
+                    dyn.apply(mutation)
+                    try property.set(value: dyn, on: &self)
+                default:
+                    break;
+                }
+            }
+        } catch {
+            fatalError("Applying operation on all properties of \((try? typeInfo(of: Target.self))?.name ?? "Unknown Type") on DynamicProperty \((try? typeInfo(of: Self.self))?.name ?? "Unknown Type") failed.")
+        }
+    }
+}
+
+
+// MARK: Dynamics Implemenation
+fileprivate extension Dynamics {
+    
+    func execute<Target>(_ operation: (Target, _ name: String) -> ()) {
+        for (name, element) in self.elements {
+            switch element {
+            case let t as Target:
+                assert((Mirror(reflecting: element).displayStyle) == .struct, "\(element.self) \(name) on Dynamics must be a struct")
+                
+                operation(t, name)
+            case let dp as DynamicProperty:
+                assert((Mirror(reflecting: element).displayStyle) == .struct, "DynamicProperty \(name) on Dynamics must be a struct")
+                
+                dp.execute(operation)
+            case let dyn as Dynamics:
+                // no assertion needed because Dynamics is defined by Apodini
+            
+                dyn.execute(operation)
+            default:
+                break;
+            }
+        }
+    }
+    
+    mutating func apply<Target>(_ mutation: (inout Target, _ name: String) -> ()) {
+        for (name, element) in self.elements {
+            switch element {
+            case var t as Target:
+                assert((Mirror(reflecting: element).displayStyle) == .struct, "\(element.self) \(name) on Dynamics must be a struct")
+    
+                mutation(&t, name)
+                self.elements[name] = t
+            case var dp as DynamicProperty:
+                assert((Mirror(reflecting: element).displayStyle) == .struct, "DynamicProperty \(name) on Dynamics must be a struct")
+                
+                dp.apply(mutation)
+                self.elements[name] = dp
+            case var dyn as Dynamics:
+                // no assertion needed because Dynamics is defined by Apodini
```

Comment:
I'd still add the assertion though. To catch it in tests when something is changed